app/src/main/java/com/ccc/nameapp/utils/DimensionUtils.kt

```diff
@@ -11,7 +11,8 @@ object DimensionUtils {
     }
 
     fun getDimensionWithScaledDensity(context: Context, @DimenRes dimenRes: Int): Float {
-        return context.resources.getDimension(dimenRes) / context.resources.displayMetrics.scaledDensity
+        return context.resources.getDimension(dimenRes) /
+                context.resources.displayMetrics.scaledDensity
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/9cbff12fc6911fd41007439107765dbec2127af9/app/src/main/java/com/ccc/nameapp/utils/DimensionUtils.kt#L15"></span>Unexpected indentation (16) (should be 12)</td>
    </tr>
  </tbody>
</table>